marconi-chain-index/src/Marconi/ChainIndex/Indexers.hs

```diff
@@ -166,11 +198,13 @@ utxoWorker_
   -> Utxo.Depth
   -> Maybe TargetAddresses
   -- ^ Target addresses to filter for
+  -> Bool
+  -- ^ Store ReferenceScript
```

Comment:
It may be worth creating a `UtxoIndexerConfig` type to group both options.
(and to pass it along as soon as we need to pass both parameters)